node/src/chain_spec.rs

```diff
@@ -99,6 +89,10 @@ fn development_genesis() -> GenesisConfig {
             .duration_since(wasm_timer::UNIX_EPOCH)
             .expect("cannot get system time for genesis")
             .as_millis() as Timestamp,
+        // Starports
+        vec![],
```

Comment:
This is causing `cargo run -- --dev --tmp --alice` to fail with panic at `pallets/cash/src/lib.rs:713`

```
Thread 'main' panicked at 'Starports must be set in the genesis config',
```

we should probably change something here .... i can look into what we were doing before

Reply:
cool, though maybe it should fail since we need a starport to get going, right?